code-of-conduct-best-practices.md

```diff
@@ -0,0 +1,51 @@
+# Best practices for interaction at denhac
+
+
+## Purpose
+
+This document outlines best practices for behavior at denhac, complementing our existing code of conduct. It emphasizes each member's responsibility to assess their intent, action, and impact. The code of conduct establishes minimum standards for acceptable conduct, identifies unacceptable behavior, and describes the procedures for reporting such behavior to the board of directors. denhac's goal is to foster a positive environment where members and visitors engage constructively, minimizing conflicts and preventing harassment.
```

Comment:
live feedback: the Best Practices should say something about how it the CoC supersedes it